src/entity-manager.js

```diff
@@ -21,35 +21,36 @@ export class EntityManager {
   }
 
   /**
-   * Register an array of entity references.
+   * Register an array of entity classes.
    *
-   * @param {Entity[]|Entity} entities Array or object of entities.
+   * @param {function[]|function} Entity classes array or object of Entity constructors.
    *
    * @return {EntityManager} this
    * @chainable
    */
-  registerEntities(entities) {
-    for (let reference in entities) {
-      if (!entities.hasOwnProperty(reference)) {
+  registerEntities(EntityClasses) {
+    for (let property in EntityClasses) {
+      if (!EntityClasses.hasOwnProperty(property)
```

Comment:
at that place it can just skip false entries silently. with registerEntity it should tell the user (or as now, let it be his own responsibility)

also `if (!EntityClasses.hasOwnProperty(property)` can only be at resisterEntities (i'm not sure what is for admirably)

given that: should i move !Entity.isPrototypeOf(EntityClasses[property])) `` to registerEntity?